src/service/fixed-layer.js

```diff
@@ -128,6 +128,32 @@ export class FixedLayer {
   }
 
   /**
+   * Apply or reset transform style to fixed elements
+   * @param {string|undefined} transform
```

Comment:
Let's make it nullable, i.e. `@param {?string} transform`. `undefined` doesn't fit very well here.